WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -1174,6 +1180,7 @@ extension UIImage {
     static var builtInReaderError: UIImage {
         return UIImage(named: "built-in-reader-error")!
     }
+
```

Comment:
Not a blocker, empty line here

Reply:
Fixed https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/3993a92f842bf44680a0a96c3e2df39f427dc002